container/agent-exec.sh

```
@@ -0,0 +1,138 @@
+  git config --global user.email "${GIT_AUTHOR_EMAIL:-omniclaw@users.noreply.github.com}"
+
+  if command -v gt &> /dev/null; then
+    if ! gt auth status &> /dev/null; then
+      gt auth --token "$GITHUB_TOKEN" 2>/dev/null || true
+    fi
+  fi
+fi
+
+# Set HOME to per-agent session dir's parent so .claude/ lands correctly
+if [ -n "$AGENT_SESSION_DIR" ]; then
+  export HOME=$(dirname "$AGENT_SESSION_DIR")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)